### DISCUSSION TITLE
2023-09-20

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#536] [daily_meeting]
### 오늘의 할 일

- 배포 : 랜딩 페이지 변경, 반응형, 에러 출력, 모달, 페이지 이동
- 프론트엔드 : 이미지 최적화
- 백엔드 : 서버 톰캣 스레드 풀 튜닝

**Comment by woosung1223** [comment]
### 체크인
6점 - 너무 졸리네여 ㅠㅠ 잠만 자고싶네요

**Comment by nlom0218** [comment]
### 체크인

7점 - 날씨가 찌뿌둥해서 저도 찌뿌등해요

**Comment by jaehee329** [comment]
### 체크인
7점 - 쏘쏘에요

**Comment by woo-jk** [comment]
### 체크인
8점 - 나쁘지 않음

**Comment by yeopto** [comment]
### 체크인

8점 - 조금 피곤하긴 하네요 오늘은

**Comment by aak2075** [comment]
### 체크인

10점 - 무난합니다

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 오늘 스레드 풀 튜닝 조져보죠

**Comment by jaehee329** [comment]
카카오 브라우저에서 구글 로그인이 불가한 이슈 관련:
https://devtalk.kakao.com/t/oauth-403/117012/2
